css/navbar.css

```diff
@@ -0,0 +1,115 @@
+/* Moved all navbar stylings out to make easier */
+
+
+/* Basic Top Nav Bar */
+#topnav-container{
+  position: - webkit-sticky; /*safari support*/
```

Comment:
Consistency nitpick between files
sometimes you have comment block outside braces, sometimes above a line, sometimes after a line

Reply:
Fixed in https://github.com/Zenloth/zenloth.github.io/commit/47a514983052c924ba3c10642f98ec546c36756f I think makes it a bit clearer. Let me know what you think.

Reply:
Slight extra tweak https://github.com/Zenloth/zenloth.github.io/commit/3597a630ea10b7b436eff06d760f5164b4c9fe81